### PR TITLE
Fix tadpole being able to land on disks

### PR DESCRIPTION
**Pull request by vvvv-vvvv**
## About The Pull Request

title

## Changelog

:cl:
fix: Fix tadpole being able to land on disks
/:cl:

### CODE REVIEW

**Comment by lbnesquik** [comment]
Long time coming, thanks.

**Comment by chizzy376** [comment]
Massive PO nerf
Jokes aside thanks for this, though you might want to move a nuke disk in Research Outpost as Canterbury can land on it, and I'm not sure about the interaction between canterbury crush and indestructible objs

File: code/game/objects/machinery/computer/nuke_disk_generator.dm [hunk 1]
```diff
@@ -216,7 +216,7 @@ GLOBAL_LIST_INIT(nuke_disk_generator_types, list(/obj/machinery/computer/nuke_di
 	desc = "Some dusty old computer. Looks non-functional"
 	density = TRUE
 	anchored = TRUE
-	resistance_flags = RESIST_ALL
+	resistance_flags = RESIST_ALL|DROPSHIP_IMMUNE
```
[the remark this answers is not in the document]
Comment (vvvv-vvvv) [reply code/game/objects/machinery/computer/nuke_disk_generator.dm:219]:
```suggestion
        resistance_flags = RESIST_ALL
```

File: code/game/objects/machinery/computer/nuke_disk_generator.dm [hunk 2]
```diff
@@ -13,7 +13,7 @@
 	interaction_flags = INTERACT_MACHINE_TGUI
 	circuit = /obj/item/circuitboard/computer/nuke_disk_generator
 
-	resistance_flags = INDESTRUCTIBLE|UNACIDABLE
+	resistance_flags = RESIST_ALL|DROPSHIP_IMMUNE
```
[the remark this answers is not in the document]
Comment (vvvv-vvvv) [reply code/game/objects/machinery/computer/nuke_disk_generator.dm:16]: it wouldve just interacted with the machine instead

**Pull request merged**